src/components/form/FToggle.vue

```diff
@@ -0,0 +1,348 @@
+<template lang="pug">
+FField.FToggle(
+  :class="classes"
+  :style="style"
+  v-bind="{ name, hint, hideHint, hintTextColor, hintIcon }"
+)
+  label.FToggle__label(:for="name" :class="labelClasses")
+    .FToggle__wrapper
+      input.FToggle__hiddenCheckbox(
+        :id="name"
+        ref="toggleRef"
+        v-model="fieldValue"
+        :name="name"
+        type="checkbox"
+        :disabled="disabled"
+        @blur="handleBlur"
+        @change="handleChange"
+        @focus="handleFocus"
+        @keypress.enter="fieldValue = !fieldValue"
+      )
+      .FToggle__track(:class="toggleClasses")
+        .FToggle__track__thumb
+          FIcon.FToggle__toggledIcon(
+            v-if="!noCheckmarkIcon"
+            name="checkmark"
+            :color="toggledColor"
+            size="10"
+            :class="iconClasses"
+          )
+    span.FToggle__labelText(v-if="label") {{ label }}
+
+    span.FToggle__labelText(v-else)
+      slot(name="label")
+</template>
+  
+<style lang="stylus">
+.FToggle__label
+  position relative
+  display flex
+  cursor pointer
+  z-index 1
+  align-items flex-start
+  gap rem(12)
+
+  &--reverse
+    flex-direction row-reverse
+    justify-content space-between
+  
+.FToggle__wrapper
+  display flex
+  position relative
+  align-items center
+
+.FToggle__track
+  width rem(44)
+  height rem(24)
+  background-color var(--FToggle--color)
+  position relative
+  border-radius rem(12)
+  elevation('inset')
+  outline-style solid
+  outline-width rem(1)
+  outline-color var(--FToggle--border-color)
+  transition outline .1s, background-color .2s
+
+  &:hover, &:focus
+    outline-width rem(2)
+    outline-color var(--FToggle--hover-border-color)
+
+  &--toggled
+    background-color var(--FToggle--toggled-color)
+    outline-color var(--FToggle--toggled-border-color)
+    .FToggle__track__thumb
+      left 'calc(100% - %s)' % rem(22)
+  
+.FToggle__track__thumb
+  position absolute
+  left rem(2)
+  top rem(2)
+  width rem(20)
+  height rem(20)
+  border-radius rem(50%)
+  transition left .2s
+  background-color var(--FToggle--thumb-color)
+  elevation(2)
+
+.FToggle__toggledIcon
+    position absolute
+    left 50%
+    top calc(50% + 1px)
+    transform translate(-50%, -50%)
+    opacity 0
+    transition opacity .1s, transform .1s
+    
+    &--visible
+      opacity 1
+
+.FToggle__hiddenCheckbox
+    opacity 0
+    width 0
+    height 0
+
+    &:focus + .FToggle__track
+      outline-width rem(2)
+      outline-color var(--FToggle--hover-border-color)
+  
+.FToggle__labelText
+    color var(--FToggle--text-color) !important
+    user-select none
+    cursor pointer
+    max-width rem(640)
```

Comment:
```suggestion
.FToggle__labelText
    color var(--FToggle--text-color) !important
    user-select none
    cursor pointer
    max-width rem(640)
    margin-top rem(1)
```

Reply:
Maybe a bit of margin-top on this one ?  With the big text, I feel like it's not centered, what do you think ? 🤔